src/axom/slic/interface/slic_macros.hpp

```diff
@@ -129,10 +145,10 @@
         std::ostringstream __oss;                                     \
         __oss << msg;                                                 \
         axom::slic::logErrorMessage(__oss.str(), __FILE__, __LINE__); \
-      }                                                               \
-      else                                                            \
-      {                                                               \
-        axom::slic::abortIfEnabled(axom::slic::message::Error);       \
+        if(axom::slic::isAbortOnErrorsEnabled())                      \
```

Comment:
This needs to about if not on root as well

Reply:
> This needs to abort if not on root as well

Yes, but I don't think that can be done until we have a better collective/non-collective solution (unless I'm missing the obvious solution here...).

Non-root ranks would have to be notified that a root rank is aborting.